src/components/Header/Header.tsx

```diff
@@ -31,43 +29,162 @@ export function Header(props: HeaderProps) {
+		}
+		setBrandTop(header.brandTop || DEFAULT_HEADER.brandTop);
+		setHomeLinkProps(header.homeLinkProps || DEFAULT_HEADER.homeLinkProps);
+		setServiceTitle(header.serviceTitle || DEFAULT_HEADER.serviceTitle);
+		setOperatorLogo(header.operatorLogo || DEFAULT_HEADER.operatorLogo);
+	}, [header]);
 
-	useEffect(
-		() => {
-			const others = header?.quickAccessItems || [];
-			setQuickAccessItems([
-				...others,
-				{
-					iconId: 'fr-icon-lock-line',
-					buttonProps: {
-						onClick: handleOidcAuth,
-					},
-					text: getAuthLabel(isAuthenticated),
+	useEffect(() => {
+		const others = header?.quickAccessItems || [];
+		setQuickAccessItems([
+			...others,
+			{
+				iconId: 'fr-icon-lock-line',
+				buttonProps: {
+					onClick: handleOidcAuth,
 				},
-			]);
-		},
-		[isAuthenticated, handleOidcAuth, header]
-	);
+				text: getAuthLabel(isAuthenticated),
+			},
+		]);
+	}, [isAuthenticated, handleOidcAuth, header]);
 
 	return (
-		<HeaderDSFR
-			brandTop={<ConvertContent content={brandTop as any} />}
-			homeLinkProps={homeLinkProps}
-			serviceTitle={serviceTitle}
-			operatorLogo={operatorLogo}
-			quickAccessItems={quickAccessItems}
-		/>
+		<header role="banner" className="fr-header" id="header">
+			<div className="fr-header__body">
+				<div className="fr-container">
+					<div className="fr-header__body-row">
+						<div className="fr-header__brand fr-enlarge-link">
+							<div className="fr-header__brand-top">
+								<div className="fr-header__logo">
+									<p className="fr-logo">
+										{<ConvertContent content={brandTop as any} />}
+									</p>
+								</div>
+								<div className="fr-header__operator">
+									<img
+										className="fr-responsive-img"
+										style={{ maxWidth: '3.5rem' }}
+										src={operatorLogo?.imgUrl}
+										alt={operatorLogo?.alt}
+									/>
+								</div>
+								<div className="fr-header__navbar">
+									<button
+										className="fr-btn--menu fr-btn"
+										data-fr-opened="false"
+										aria-controls="modal-577"
+										aria-haspopup="menu"
+										id="button-578"
+										title="Menu"
+									>
+										Menu
+									</button>
+								</div>
+							</div>
+							<div className="fr-header__service">
+								<a
+									href={homeLinkProps.href}
+									title={`Accueil - ${homeLinkProps.title} - République Française`}
+								>
+									<p className="fr-header__service-title">{serviceTitle}</p>
+								</a>
+							</div>
+						</div>
+						<div className="fr-header__tools">
+							<div className="fr-header__tools-links">
+								<ul className="fr-btns-group">
+									<li>
+										<button
+											className="fr-btn fr-fi-theme-fill fr-link--icon-left"
+											aria-controls="fr-theme-modal"
+											data-fr-opened="false"
+										>
+											Paramètres d'affichage
+										</button>
+									</li>
+									{quickAccessItems &&
+										quickAccessItems.map(function (quickAccessItem, index) {
```

Comment:
There was an arror on this line and line 163 when I did a `yarn start` on local. 
I was incited to change to an arrow function
i.e. instead of `xyz.map(function(abc) { ... }` use `xyz.map((abc)=>{ ... }`
I will make the changes and repush to see if it fixes our problem with the gh actions.